rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
Sorry for the delay in getting around to this. It does look like the plugin got stricter about lints.

For Support-v4:

```
<?xml version="1.0" encoding="UTF-8"?>
<issues format="4" by="lint 25.1.7">

    <issue
        id="ResourceType"
        severity="Fatal"
        message="Expected resource of type id"
        category="Correctness"
        priority="7"
        summary="Wrong Resource Type"
        explanation="Ensures that resource id&apos;s passed to APIs are of the right type; for example, calling `Resources.getColor(R.string.name)` is wrong."
        errorLine1="    swipeRefreshLayout.setId(1);"
        errorLine2="                             ~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-support-v4/src/androidTest/java/com/jakewharton/rxbinding/support/v4/widget/RxSwipeRefreshLayoutTestActivity.java"
            line="29"
            column="30"/>
    </issue>

</issues>
```

And design. Design is weirder because those themes are definitely not private unless I'm missing something

```
    <issue
        id="ResourceType"
        severity="Fatal"
        message="Expected resource of type id"
        category="Correctness"
        priority="7"
        summary="Wrong Resource Type"
        explanation="Ensures that resource id&apos;s passed to APIs are of the right type; for example, calling `Resources.getColor(R.string.name)` is wrong."
        errorLine1="    view.setId(1);"
        errorLine2="               ~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxSwipeDismissBehaviorTestActivity.java"
            line="17"
            column="16"/>
    </issue>

    <issue
        id="PrivateResource"
        severity="Warning"
        message="The resource `@style/Theme_AppCompat` is marked as private in com.android.support:appcompat-v7"
        category="Correctness"
        priority="3"
        summary="Using private resources"
        explanation="Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.

To fix this, copy the resource into your own project instead."
        errorLine1="  private final Context context = new ContextThemeWrapper(rawContext, R.style.Theme_AppCompat);"
        errorLine2="                                                                      ~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxAppBarLayoutTest.java"
            line="30"
            column="71"/>
    </issue>

    <issue
        id="PrivateResource"
        severity="Warning"
        message="The resource `@style/Theme_AppCompat` is marked as private in com.android.support:appcompat-v7"
        category="Correctness"
        priority="3"
        summary="Using private resources"
        explanation="Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.

To fix this, copy the resource into your own project instead."
        errorLine1="  private final Context context = new ContextThemeWrapper(rawContext, R.style.Theme_AppCompat);"
        errorLine2="                                                                      ~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxNavigationViewTest.java"
            line="27"
            column="71"/>
    </issue>

    <issue
        id="PrivateResource"
        severity="Warning"
        message="The resource `@style/Theme_AppCompat` is marked as private in com.android.support:appcompat-v7"
        category="Correctness"
        priority="3"
        summary="Using private resources"
        explanation="Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.

To fix this, copy the resource into your own project instead."
        errorLine1="  private final Context context = new ContextThemeWrapper(rawContext, R.style.Theme_AppCompat);"
        errorLine2="                                                                      ~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxSnackbarTest.java"
            line="25"
            column="71"/>
    </issue>

    <issue
        id="PrivateResource"
        severity="Warning"
        message="The resource `@style/Theme_AppCompat` is marked as private in com.android.support:appcompat-v7"
        category="Correctness"
        priority="3"
        summary="Using private resources"
        explanation="Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.

To fix this, copy the resource into your own project instead."
        errorLine1="  private final Context context = new ContextThemeWrapper(rawContext, R.style.Theme_AppCompat);"
        errorLine2="                                                                      ~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxTabLayoutTest.java"
            line="30"
            column="71"/>
    </issue>

    <issue
        id="PrivateResource"
        severity="Warning"
        message="The resource `@style/Theme_AppCompat` is marked as private in com.android.support:appcompat-v7"
        category="Correctness"
        priority="3"
        summary="Using private resources"
        explanation="Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.

To fix this, copy the resource into your own project instead."
        errorLine1="  private final Context context = new ContextThemeWrapper(rawContext, R.style.Theme_AppCompat);"
        errorLine2="                                                                      ~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="/Users/zsweers/dev/android/contributing/RxBinding/rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxTextInputLayoutTest.java"
            line="23"
            column="71"/>
    </issue>
```